components/tools/OmeroWeb/omeroweb/webclient/views.py

```diff
@@ -2877,8 +2878,10 @@ def get_original_file(request, fileId, download=False, conn=None, **kwargs):
 
     orig_file = conn.getObject("OriginalFile", fileId)
     if orig_file is None:
-        return handlerInternalError(
-            request, "Original File does not exists (id:%s)." % (fileId))
+        rsp = ConnCleaningHttpResponse(StringIO(
```

Comment:
Sorry - coming to this a bit late...
As I understood it, we're using ```ConnCleaningHttpResponse``` in places where we don't want to call ```conn.close()``` until the response is closed (because we're downloading a file etc), but in these cases the response is not downloading anything so we could simply do this?
```
conn.close(hard=False)
return HttpResponseNotFound("Original File does not exist (id:%s)." % (fileId))
```

Reply:
Yes we could.

I was preferring the usage of the same response type and keeping all the close logic in one place.

Reply:
OK, cool. I was really just checking that I understood the logic.